_developer-files/connect/api/objects/form-properties/sources/databases/hp-mysql-object.md

```diff
@@ -0,0 +1,139 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/database-source-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+## PLEASE REMOVE COMMENTS WHEN FINISHED
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+product-type: "connect"
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-hp-mysql-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "MySQL (HP) Source Form Property"
+api-type: "platform.hp-mysql"
+display-name: "MySQL (HP)"
+
+source-type: "database"
+docs-name: "mysql"
+db-type: "mysql"
+
+property-description: |
+  MySQL databases
+
+description: |
+  **Note**: This version of the MySQL source differs from the version used by the `platform.mysql` form property. Refer to the [{{ form-property.display-name }} integration feature summary]({{ doc-link | append: "#feature-summary" }}) for more info.
+
+
+# -------------------------- #
+#      OBJECT ATTRIBUTES     #
+# -------------------------- #
+
+uses-common-fields: true
+uses-feature-fields: true
+uses-start-date: true
+
+object-attributes:
```

Comment:
- `allow_non_auto_increment_pks` - It doesn't look like this is in the report card for `tap.hp-mysql`. I'm not sure if this is intentional or not, but it should be confirmed with the sources team and then removed if needed

- `ssh`-related properties, `server_id`, `ssl_client_auth_enabled` are all displaying twice. I think this is because you have them listed out here, but they don't need to be. The `uses-feature-fields` flag being set to `true` should make it so that you don't need to do that.